mars_api/data_import/data_importer.py

```diff
@@ -0,0 +1,134 @@
+from typing import Dict
+
+import colorlog
+from dateutil import parser
+from django.db.utils import IntegrityError
+
+from mars_api.serializers import GameSerializerForImportedData, PlayerScoreSerializer
+
+
+def str_to_bool(arg):
+    return bool(int(arg))
+
+
+GAME_FIELD_MAPPING = {
+    (0, "id", int),
+    (1, "date", parser.parse),  # datetime
+    (2, "game_map", str),
+    (3, "prelude", str_to_bool),
+    (4, "colonies", str_to_bool),
+    (5, "number_of_generations", int),
+}
+
+GAME_VALUE_REMAPPING = {"Default": "Tharsis"}
```

Comment:
Game value or a map?

Reply:
`VALUE` as in remapping the values in the game dict + a more generic naming just in case more values need to be remapped in the future.